<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToDoList</class>
 <widget class="QMainWindow" name="ToDoList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.00526316 rgba(213, 229, 255, 255), stop:1 rgba(255, 247, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="InputLine">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(11, 130, 2);
background-color: rgb(255, 253, 221);</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(163, 23, 5);
background-color: rgb(255, 253, 221);</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210, 255, 206)</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(193, 0, 0);
background-color: rgb(255, 203, 194)</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListHolder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(90, 90, 90);
background-color: rgb(255, 253, 221);</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(227, 129, 16);
background-color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(227, 129, 16);
background-color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <zorder>ListHolder</zorder>
   <zorder>InputLine</zorder>
   <zorder>AddButton</zorder>
   <zorder>RemoveButton</zorder>
   <zorder>SaveButton</zorder>
   <zorder>ClearButton</zorder>
   <zorder>EditButton</zorder>
   <zorder>UpButton</zorder>
   <zorder>DownButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
